packages/elpi-option-legacy-parser/elpi-option-legacy-parser.1.15.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
```

Comment:
I think it would be nice to not version this package according to the version of elpi. What about just having `elpi-option-legacy-parser.1` like other conf packages?

Reply:
done